packages/sdk-btc/source/transaction.service.ts

```diff
@@ -170,9 +162,12 @@ export class TransactionService extends Services.AbstractTransactionService {
 		psbt.finalizeAllInputs();
 	}
 
-	#derivationMethod(
-		derivationMethod: "bip39" | "bip44" | "bip49" | "bip84",
-	): (publicKey: string, network: string) => string {
+	#addressingSchema(levels: Levels): "bip44" | "bip49" | "bip84" {
+		// @ts-ignore
+		return `bip${levels.purpose}`;
```

Comment:
You can extract `"bip44" | "bip49" | "bip84"` into something like `type BipLevel = "bip44" | "bip49" | "bip84";` and then cast the return to `BipLevel` to avoid the ignore.